coverage/main.go

```diff
@@ -0,0 +1,27 @@
+/*
+Package main calculates and stores code coverage information for given project.
+*/
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+const defaultCoverageProfileName = "coverage_profile.txt"
+const defaultCoverageTargetDir = "./target_pkg/"
+
+func main() {
+
+	fmt.Println("entering code coverage main")
+	artifactsDir := flag.String("artifacts", "./artifacts", "directory for artifacts")
+	coverageTargetDir := flag.String("cov-target", defaultCoverageTargetDir, "target directory for test coverage")
+	profileName := flag.String("profileName", defaultCoverageProfileName, "file name for coverage profile")
+	pullNumFlag := flag.String("pull-number", "", "PR number")
```

Comment:
Since you're going to want to merge the code into the target branch, etc -- you can import the `downwardapi` struct and use that, from `$JOB_SPEC`?